content/tutorial/tutorial.md

```diff
@@ -321,44 +326,44 @@ class Square extends React.Component {
 
-## Completing the Game {#completing-the-game}
+## खेल को पूरा करना {#completing-the-game}
 
-We now have the basic building blocks for our tic-tac-toe game. To have a complete game, we now need to alternate placing "X"s and "O"s on the board, and we need a way to determine a winner.
+अब हमारे पास टिक-टैक-टो गेम के लिए बुनियादी बिल्डिंग ब्लॉक हैं। एक पूरा गेम बनाने के लिए, अब हमें वैकल्पिक रूप से Board पर "X" और "O" को रखने की आवश्यकता है, और हमें विजेता को निर्धारित करने के लिए एक तरीका चाहिए।
 
-### Lifting State Up {#lifting-state-up}
+### लिफ्टिंग state अप {#lifting-state-up}
 
-Currently, each Square component maintains the game's state. To check for a winner, we'll maintain the value of each of the 9 squares in one location.
+वर्तमान में, प्रत्येक Square कौम्पोनॅन्ट खेल की स्थिति को बनाए रखता है। विजेता की जाँच करने के लिए, हम एक स्थान पर 9 वर्गों में से प्रत्येक का मान बनाए रखेंगे।
 
-We may think that Board should just ask each Square for the Square's state. Although this approach is possible in React, we discourage it because the code becomes difficult to understand, susceptible to bugs, and hard to refactor. Instead, the best approach is to store the game's state in the parent Board component instead of in each Square. The Board component can tell each Square what to display by passing a prop, [just like we did when we passed a number to each Square](#passing-data-through-props).
+हम सोच सकते हैं कि Board को Square के state के लिए प्रत्येक Square से पूछना चाहिए। हालांकि यह दृष्टिकोण React में संभव है, हम इसे हतोत्साहित करते हैं क्योंकि कोड को समझना मुश्किल हो जाता है, कीड़े के लिए अतिसंवेदनशील और रिफ्लेक्टर के लिए कठिन। इसके बजाय, सबसे अच्छा तरीका यह है कि खेल के राज्य को प्रत्येक Square के बजाय पैरेंट Board कौम्पोनॅन्ट में स्टोर किया जाए। Board कौम्पोनॅन्ट प्रत्येक Square को props पास करके यह बता सकता है कि किसी प्रस्ताव को पारित करके क्या प्रदर्शित करना है, [ठीक उसी तरह जैसे हमने प्रत्येक Square के लिए एक नंबर पारित किया था](#passing-data-through-props)।
 
-**To collect data from multiple children, or to have two child components communicate with each other, you need to declare the shared state in their parent component instead. The parent component can pass the state back down to the children by using props; this keeps the child components in sync with each other and with the parent component.**
+**कई चिल्ड्रन से डेटा एकत्र करने के लिए, या दो चाइल्ड कौम्पोनॅन्ट को एक दूसरे के साथ संवाद करने के लिए, आपको इसके बजाय उनके पेरेंट कौम्पोनॅन्ट में शेयर्ड state को घोषित करने की आवश्यकता है। पेरेंट कौम्पोनॅन्ट props का उपयोग करके state को वापस चिल्ड्रन को दे सकते हैं; यह चाइल्ड कौम्पोनॅन्ट को एक दूसरे के साथ और पेरेंट कौम्पोनॅन्ट के साथ तालमेल में रखता है।**
 
-Lifting state into a parent component is common when React components are refactored -- let's take this opportunity to try it out.
+React कौम्पोनॅन्ट के अपवर्तित होने पर पेरेंट कौम्पोनॅन्ट में भारोत्तोलन की स्थिति सामान्य है -- आइए इस अवसर पर इसको आजमाएं। 
```

Comment:
अपवर्तित ?

Reply:
`React कॉम्पोनेन्ट रीफैक्टर होने पर state को पैरेंट कॉम्पोनेन्ट में रखना सामान्य है -- आइए इसको आजमाएं।` ?

Reply:
@saranshkataria Any input on this?

Reply:
cool